azure-pipelines-official.yml

```diff
@@ -75,6 +75,8 @@ extends:
       name: $(DncEngInternalBuildPool)
       image: windows.vs2022preview.amd64
       os: windows
+    settings:
+      networkIsolationPolicy: Permissive,CFSClean
```

Comment:
What is the default when this is not set?

Why is it not sufficient to only update the tests to not use nuget.org?